<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfigWidget</class>
 <widget class="QWidget" name="GameConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="mp_layout_4">
     <item row="4" column="1">
      <widget class="QLineEdit" name="mp_edit_elf">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="mp_layout_5">
       <item>
        <widget class="QLabel" name="mp_label_opl_version">
         <property name="text">
          <string>OPL Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mp_combo_opl_version"/>
       </item>
       <item>
        <spacer name="mp_spacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mp_label_game_id">
       <property name="text">
        <string>Game ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="mp_btn_id_from_game">
       <property name="text">
        <string>Get from the Game</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/restore</normaloff>:/images/restore</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mp_edit_game_id">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mp_label_elf">
       <property name="text">
        <string>Custom ELF</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="mp_line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mp_layout_2">
     <item>
      <widget class="QGroupBox" name="mp_groupbox_modes">
       <property name="title">
        <string>Modes</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_1">
          <property name="text">
           <string>Mode 1 ― Accurate Reads</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_2">
          <property name="text">
           <string>Mode 2 ― Synchronous Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_3">
          <property name="text">
           <string>Mode 3 ― Unhook Syscalls</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_5">
          <property name="text">
           <string>Mode 5 ― Emulate DVD-DL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_6">
          <property name="text">
           <string>Mode 6 ― Disable IGR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mp_checkbox_mode_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mp_layout_3">
       <item>
        <widget class="QGroupBox" name="mp_groupbox_vmc0">
         <property name="title">
          <string>Virtual Memory Card #1</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="mp_combo_vmc0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAlphabetically</enum>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_btn_create_vmc0">
            <property name="text">
             <string>Create VMC</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/images/add</normaloff>:/images/add</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mp_groupbox_vmc1">
         <property name="title">
          <string>Virtual Memory Card #2</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="mp_combo_vmc1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAlphabetically</enum>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_btn_create_vmc1">
            <property name="text">
             <string>Create VMC</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/images/add</normaloff>:/images/add</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="mp_spacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mp_groupbox_gsm">
     <property name="title">
      <string>Graphic Synthesiser Mode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="mp_layout_6">
        <item>
         <widget class="QRadioButton" name="mp_radio_disable_gsm">
          <property name="text">
           <string>Disable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mp_radio_enable_gsm">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mp_radio_use_global_gsm">
          <property name="text">
           <string>Use Global Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="mp_spacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="mp_groupbox_gsm_settings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="mp_layout_7">
           <item>
            <widget class="QLabel" name="mp_label_video_mode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Video Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="mp_combo_video_mode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="mp_checkbox_skip_fmv">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Skip Full Motion Videos</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="mp_layout_8">
           <item>
            <widget class="QLabel" name="mp_label_hpos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Horizontal Position</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mp_spinbox_hpos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>-2000</number>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mp_label_vpos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Vertical Position</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mp_spinbox_vpos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>-2000</number>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="mp_checkbox_emulate_field_flipping">
           <property name="text">
            <string>Emulate FIELD Flipping</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="mp_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mp_layout">
     <item>
      <spacer name="mp_spacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_delete">
       <property name="text">
        <string>&amp;Delete Configuration</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_clear">
       <property name="text">
        <string>&amp;Clear Configuration</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/restore</normaloff>:/images/restore</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_save">
       <property name="text">
        <string>&amp;Save Configuration</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/save</normaloff>:/images/save</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_combo_opl_version</tabstop>
  <tabstop>mp_edit_game_id</tabstop>
  <tabstop>mp_btn_id_from_game</tabstop>
  <tabstop>mp_edit_elf</tabstop>
  <tabstop>mp_checkbox_mode_1</tabstop>
  <tabstop>mp_checkbox_mode_2</tabstop>
  <tabstop>mp_checkbox_mode_3</tabstop>
  <tabstop>mp_checkbox_mode_4</tabstop>
  <tabstop>mp_checkbox_mode_5</tabstop>
  <tabstop>mp_checkbox_mode_6</tabstop>
  <tabstop>mp_checkbox_mode_7</tabstop>
  <tabstop>mp_checkbox_mode_8</tabstop>
  <tabstop>mp_combo_vmc0</tabstop>
  <tabstop>mp_btn_create_vmc0</tabstop>
  <tabstop>mp_combo_vmc1</tabstop>
  <tabstop>mp_btn_create_vmc1</tabstop>
  <tabstop>mp_radio_disable_gsm</tabstop>
  <tabstop>mp_radio_enable_gsm</tabstop>
  <tabstop>mp_radio_use_global_gsm</tabstop>
  <tabstop>mp_combo_video_mode</tabstop>
  <tabstop>mp_checkbox_skip_fmv</tabstop>
  <tabstop>mp_spinbox_hpos</tabstop>
  <tabstop>mp_spinbox_vpos</tabstop>
  <tabstop>mp_checkbox_emulate_field_flipping</tabstop>
  <tabstop>mp_btn_delete</tabstop>
  <tabstop>mp_btn_clear</tabstop>
  <tabstop>mp_btn_save</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
